evergreen.yaml

```diff
@@ -115,49 +115,37 @@ task_groups:
 #           Buildvariants             #
 #######################################
 buildvariants:
-  - name: race-detector
-    display_name: Race Detector (Arch Linux)
-    expansions:
-      RACE_DETECTOR: true
-      GOROOT: /opt/golang/go1.16
-      TEST_TIMEOUT: 1h
-    run_on:
-      - archlinux-new-large
-    tasks: [ "testGroup" ]
-
   - name: lint
-    display_name: Lint (Arch Linux)
+    display_name: Lint
     expansions:
-      GOROOT: /opt/golang/go1.16
+      GOROOT: /opt/golang/go1.20
     run_on:
-      - archlinux-new-small
+      - ubuntu2204-small
     tasks:
       - lintGroup
       - verify-mod-tidy
 
   - name: ubuntu
-    display_name: Ubuntu 18.04
+    display_name: Ubuntu 22.04
     expansions:
-      GOROOT: /opt/golang/go1.16
+      GOROOT: /opt/golang/go1.20
+      RACE_DETECTOR: true
```

Comment:
Do we always want to run the race detector on this? Should we have a variant that runs the same tests without it?

Reply:
Why would we need a variant that does the same tests without it? We need to run the race detector on some variant and that seems like it would just duplicate tests?

Reply:
Yeah I thought it was because running the tests with the race detector would be slower (we separate it out for Evergreen but not Cedar). I think this is fine then.